attributes rule gives violation report with @testable

### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Bug Report

The attributes rule seems to give wrong results with `@testable`. In the sample code to reproduce below, if you remove the blank line between `@testable` and `@available` the violation warning goes away, but that is wrong. `@testable` is not an attribute of the class and is correctly separated by vertical white space.

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint --config swiftlint.yml
Loading configuration from 'swiftlint.yml'
Linting Swift files in current working directory
Linting 'DeleteMeTests.swift' (1/1)
/Users/bestave/Desktop/DeleteMe/DeleteMeFP/AttributeTest/DeleteMeTests.swift:13:1: warning: Attributes Violation: Attributes should be on their own lines in functions and types, but on the same line as variables and imports. (attributes)
Done linting! Found 1 violation, 0 serious in 1 file.
```

### Environment

* SwiftLint version: 0.25.1 (release and latest source)
* Installation method used: SwiftLint.pkg installer and also cloned/built from master source
* Paste your configuration file:

```yml
opt_in_rules:
  - attributes
```

* Are you using [nested configurations]? No
* Which Xcode version are you using? 9.3.1
* Do you have a sample that shows the issue? Yes:

```swift
import XCTest
@testable import DeleteMe

@available (iOS 11.0, *)
class DeleteMeTests: XCTestCase {
}
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -102,6 +102,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#2187](https://github.com/realm/SwiftLint/issues/2187)
 
+* Fix false positives in `attributes` rule when `@testable` is used.  
+  [Marcelo Fabri](https://github.com/marcelofabri)
+  [#2211](https://github.com/realm/SwiftLint/issues/2211)
+
 ## 0.25.1: Lid Locked
 
 This is the last release to support building with Swift 3.2 and Swift 3.3.

--- a/Rules.md
+++ b/Rules.md
@@ -429,6 +429,15 @@ private struct DefaultError: Error {}
 private let bar = 1
 ```
 
+```swift
+import XCTest
+@testable import DeleteMe
+
+@available (iOS 11.0, *)
+class DeleteMeTests: XCTestCase {
+}
+```
+
 </details>
 <details>
 <summary>Triggering Examples</summary>

--- a/Source/SwiftLintFramework/Rules/AttributesRule.swift
+++ b/Source/SwiftLintFramework/Rules/AttributesRule.swift
@@ -209,7 +209,7 @@ public struct AttributesRule: ASTRule, OptInRule, ConfigurationProviderRule {
                     return false
                 }
 
-                return ["func", "var", "let"].contains(keyword)
+                return ["func", "var", "let", "import"].contains(keyword)
             }
 
             guard nonAttributeTokens.isEmpty else {

--- a/Source/SwiftLintFramework/Rules/AttributesRuleExamples.swift
+++ b/Source/SwiftLintFramework/Rules/AttributesRuleExamples.swift
@@ -38,7 +38,15 @@ internal struct AttributesRuleExamples {
         "func increase(f: @autoclosure () -> Int) -> Int",
         "func foo(completionHandler: @escaping () -> Void)",
         "private struct DefaultError: Error {}",
-        "@testable import foo\n\nprivate let bar = 1"
+        "@testable import foo\n\nprivate let bar = 1",
+        """
+        import XCTest
+        @testable import DeleteMe
+
+        @available (iOS 11.0, *)
+        class DeleteMeTests: XCTestCase {
+        }
+        """
     ]
 
     static let triggeringExamples = [

